apps/web-connect/src/features/checkout/components/PromoCodeRow.tsx

```diff
@@ -21,6 +21,17 @@ export function PromoCodeRow() {
 
     const [promo, setPromo] = useState(false);
 
+    //pull promo code from 
```

Comment:
I'm going to defer to @CryptITAustria here, but the only question I have is if we should wrap this in a useEffectHook with the query string as a dependency.

Otherwise, it looks good to me!